.github/workflows/platform-ci.yml

```diff
@@ -133,7 +133,7 @@ jobs:
             coverage report
           fi
           if [ -n "$WITH_RAY" ]; then
-            pytest $PYTEST_CONFIG --cov-config .coveragerc-threaded mars/ray
+            pytest $PYTEST_CONFIG --cov-config .coveragerc-threaded mars/ray mars/oscar/backends/ray
```

Comment:
Shall us use -k option? Refer to https://docs.pytest.org/en/stable/example/markers.html#using-k-expr-to-select-tests-based-on-their-name